global_helpers/panther_greynoise_helpers.py

```diff
@@ -72,7 +96,16 @@ def subscription_level(self):
         return self.sublevel
 
     def ip_address(self, match_field) -> str:
-        return deep_get(self.noise, match_field, "ip")
+        ip_get_call = deep_get(self.noise, match_field, "ip")
```

Comment:
What happens when this returns None?

Reply:
hmm..I should account for this in the exception handling, or rather there should be a check here for ip_get_call == None

Reply:
The old code just returned None , so we want that. Add a test for this

Reply:
roger that